fastlane/Fastfile

```diff
@@ -28,6 +28,7 @@ lane :lint_source do
   #
   exclude_dirs = ["\.bundle", "*/vendor/bundle"]
   ensure_no_debug_code(text: "binding.pry", extension: ".rb", exclude: "playground.rb", exclude_dirs: exclude_dirs) # debugging code
```

Comment:
Now that I think about it, we could write

```
ensure_no_debug_code(text: "binding\.(irb|pry)", extension: ".rb", exclude: "playground.rb", exclude_dirs: exclude_dirs)
```
that way we will run a single call. No?
instead

Reply:
@lacostej I thought it would be easier to understand if explicitly separated, but the context is the same and no problem to describe them together, so fixed